src/main/java/com/nike/cerberus/domain/environment/CertificateInformation.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.domain.environment;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.List;
+
+public class CertificateInformation {
+
+    private String identityManagementCertificateName;
+    private DateTime notBefore;
+    private DateTime notAfter;
+    private DateTime uploaded;
+    private String commonName;
+    private List<String> subjectAlternateNames;
+
+    public String getIdentityManagementCertificateName() {
+        return identityManagementCertificateName;
+    }
+
+    public void setIdentityManagementCertificateName(String identityManagementCertificateName) {
```

Comment:
Is this just the name? Can we store the ARN here as well or instead?

Reply:
It just the name, as that is all that is used

Reply:
Your right, I missed that there was a conversion in the old code, fixed!